tests/pipeline/test_downsampling.py

```diff
@@ -691,40 +691,23 @@ def test_errors_on_bad_downsample_frequency(self):
         self.assertEqual(str(e.exception), expected)
 
 
-class TestDownsampledRowwiseOperation(WithAssetFinder, ZiplineTestCase):
+class TestDownsampledRowwiseOperation(WithSeededRandomPipelineEngine,
+                                      ZiplineTestCase):
     T = partial(pd.Timestamp, tz='utc')
-    START_DATE = T('2014-01-01')
-    END_DATE = T('2014-02-01')
+    START_DATE = T('2014-01-02')
```

Comment:
It's being used by the seeded random loader fixture, but I'd be fine with ditching that in favor of a manual engine setup.